homeassistant/util/dt.py

```diff
@@ -41,6 +41,18 @@ def utcnow():
     return dt.datetime.now(UTC)
 
 
+def beatsnow():
```

Comment:
Suggest that you separate your commit to bug fixes in one and new functionality in another. :smiley: